Keas.Mvc/Controllers/PersonAdminController.cs

```diff
@@ -159,6 +167,245 @@ await _context.WorkstationAssignments.AnyAsync(a => a.PersonId == person.Id))
+                                {
+                                    importResult.ErrorMessage.Add(validationResult.ErrorMessage);
+                                }
+                            }
+                        }
+                        catch (ValidationException e)
+                        {
+                            importResult.ErrorMessage.Add("Validation Exception for this row.");
+                        }
+
+                        if (importResult.ErrorMessage.Count > 0)
+                        {
+                            importResult.Success = false;
+                        }
+                        else
+                        {
+                            if (await _context.People.AnyAsync(a => a.Active && a.UserId.Equals(r.KerbId.ToLower()) && a.TeamId == team.Id))
+                            {
+                                importResult.Messages.Add($"KerbId {r.KerbId} Already active in team, no changes made.");
+                                importResult.Success = false;
+                            }
+                        }
+
+
+                        if (importResult.Success)
+                        {
+                            Person person = null;
+                            try
+                            {
+                                var personResult = await _identityService.GetOrCreatePersonFromKerberos(r.KerbId, team.Id, team, userName, userIdentity, "CSV People Import");
+                                person = personResult.Person;
+
+                                await PopulatePerson(r, person, importResult, team);
+
+                                await _context.SaveChangesAsync();
+                            }
+                            catch (Exception)
+                            {
+                                person = null;
+                                importResult.Success = false;
+                                importResult.ErrorMessage.Add($"!!!!!!!!!!!!!THERE IS A PROBLEM WITH KerbUser {r.KerbId} PLEASE CONTACT PEAKS HELP with this User ID.!!!!!!!!!!!!!!!");
+                            }
+
+                            if (person == null)
+                            {
+                                importResult.Success = false;
+                                importResult.ErrorMessage.Add($"KerbId not found.");
+                            }
+                        }
+
+
+                        resultsView.Add(importResult);
+                    }
+                }
+                catch (Exception e)
+                {
+                    counter++;
+                    var importResult = new PeopleImportResult(new PeopleImport());
+                    importResult.Success = false;
+                    importResult.LineNumber = counter;
+                    importResult.ErrorMessage.Add("There is a problem with this row preventing it from being imported. Maybe an invalid date format. Import Halted.");

```

Comment:
Isn't this exception run once per import and not a row problem?  it doesn't seem to be within the for loop

Reply:
@srkirkland It happens in the actual forEach(var r in records) it happens when a row in records isn't valid. I couldn't figure out a way to trap it per row and actually continue.